motr/client.c

```diff
@@ -928,6 +928,16 @@ int32_t m0_rc(const struct m0_op *op)
 }
 M0_EXPORTED(m0_rc);
 
+int m0_client_calculate_pi(const struct m0_obj *obj,
+		m0_bindex_t data_unit_offset,
+		char *pi_digest,
+		void *buff_addr,
+		m0_bcount_t buf_size)
+{
+	return 0;
+}
+M0_EXPORTED(m0_client_calculate_pi);
+
```

Comment:
There should be an option to pass checksum algo as well, if we move from md5sum to crc, it should be smootly done.

Reply:
Done.